<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocalServerOutputSettingsDialog</class>
 <widget class="QDialog" name="LocalServerOutputSettingsDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>103</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Font:</string>
     </property>
     <property name="buddy">
      <cstring>fontComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFontComboBox" name="fontComboBox">
     <property name="toolTip">
      <string>Choose the text font</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="fontFilters">
      <set>QFontComboBox::MonospacedFonts</set>
     </property>
     <property name="currentFont">
      <font>
       <family>Andale Mono</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Colors:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
     <item>
      <widget class="ColorPickButton" name="backgroundColor">
       <property name="toolTip">
        <string>Choose the background color</string>
       </property>
       <property name="text">
        <string>&amp;Background</string>
       </property>
       <property name="chosenColor" stdset="0">
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ColorPickButton" name="textColor">
       <property name="toolTip">
        <string>Choose the text color</string>
       </property>
       <property name="text">
        <string>&amp;Text</string>
       </property>
       <property name="chosenColor" stdset="0">
        <color>
         <red>0</red>
         <green>192</green>
         <blue>0</blue>
        </color>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorPickButton</class>
   <extends>QPushButton</extends>
   <header>colorpickbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>fontComboBox</tabstop>
  <tabstop>backgroundColor</tabstop>
  <tabstop>textColor</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LocalServerOutputSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LocalServerOutputSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
